src/sentry/models/file.py

```diff
@@ -109,7 +109,7 @@ def get_storage(config=None):
     return storage(**options)
 
 
-@control_silo_model
+@all_silo_model
```

Comment:
For files, and blobs we'll likely need to build new APIs that are organization scoped.

Getting customers to use the new APIs will be challenging though as their usecases will likely be embedded in release tooling.